lib/components/FileAttachment/index.jsx

```diff
@@ -0,0 +1,145 @@
+import React, { useRef } from "react";
+
+import { saveAs } from "file-saver";
+
+import directUploadsApi from "apis/direct_uploads";
+import { DIRECT_UPLOAD_ENDPOINT } from "common/constants";
+import Button from "components/Common/Button";
+import useUppyUploader from "hooks/useUppyUploader";
+
+import Attachment from "./Attachment";
+import {
+  DEFAULT_UPPY_CONFIG,
+  DROP_DOWN_OPTIONS_KEYS,
+  UPPY_UPLOAD_CONFIG,
+} from "./constant";
+
+import Progress from "../Editor/MediaUploader/Progress";
+
+const FileAttachment = ({
+  endpoint = DIRECT_UPLOAD_ENDPOINT,
+  attachments = [],
+  onChange = _ => {},
+}) => {
+  const fileInputRef = useRef(null);
+  const uppyConfig = { ...DEFAULT_UPPY_CONFIG };
```

Comment:
What is the point of destructuring the config here? What is even the point of assigning it a variable? Directly pass it to the hook.